### ISSUE TITLE
边缘节点加入集群报错

### ISSUE THREAD

**Issue by kkkkeynoted** [kubeedge/kubeedge#5246]
**What happened**:
边缘节点第一次尝试加入集群时报错
![图片1](https://github.com/kubeedge/kubeedge/assets/149750370/6ef29a70-f56b-483b-a3d4-7240cee7d399)

尝试使用如下命令解决，并重新加入集群
sudo mkdir /sys/fs/cgroup/systemd
sudo mount -t cgroup -o none,name=systemd cgroup /sys/fs/cgroup/systemd
边缘节点第二次尝试加入集群时报错
![图片2](https://github.com/kubeedge/kubeedge/assets/149750370/71211c02-fad1-4242-981e-68e01cb1c176)
日志如下
![图片3](https://github.com/kubeedge/kubeedge/assets/149750370/e2a29c3e-6287-4174-8cd6-d6ff3b92c48f)

**Environment**:
- Kubernetes version (use `kubectl version`):1.26.7
- KubeEdge version(e.g. `cloudcore --version` and `edgecore --version`):1.15.0
- <details><summary>Cloud nodes Environment:</summary>containerd 1.6.5

  - Hardware configuration (e.g. `lscpu`):
  - OS (e.g. `cat /etc/os-release`):
  - Kernel (e.g. `uname -a`):
  - Go version (e.g. `go version`):
  - Others:

  </details>
- <details><summary>Edge nodes Environment:</summary>

  - edgecore version (e.g. `edgecore --version`):
  - Hardware configuration (e.g. `lscpu`):
  - OS (e.g. `cat /etc/os-release`):
  - Kernel (e.g. `uname -a`):
  - Go version (e.g. `go version`):
  - Others:

  </details>

**Comment by Shelley-BaoYue** [comment]
Could you please provider the complete edgecore logs,  I would like to see the error messages before the logs as below.
![image](https://github.com/kubeedge/kubeedge/assets/48788150/76ad6803-6626-499b-b6e2-6aa901549f7a)

**Comment by kkkkeynoted** [comment]
This is the beginning of the edgecore log，I'm not sure if there's any information you want to see
![图片4](https://github.com/kubeedge/kubeedge/assets/149750370/d5ab6a5d-114f-4cc2-9553-9335835ee0f0)

**Comment by Shelley-BaoYue** [comment]
It seems edgecore didn't register node to cloud. You can run `cat /etc/kubeedge/config/edgecore.yaml |grep register` to see if registerNode is enable.  It would be better if you provide the whole file of edgecore.yaml

**Comment by kkkkeynoted** [comment]
@Shelley-BaoYue Yes, edge node was not successfully registered in cloud, The error in the log is" “Nameserver limits exceeded” err=“Nameserver limits were exceeded, some nameservers have been omitted, the applied nameserver line is......" . I solved the problem based on the methods provided online, but did not achieve the desired effect.
This is the result of executing the command  “cat /etc/kubeedge/config/edgecore.yaml |grep register”
![图片1](https://github.com/kubeedge/kubeedge/assets/149750370/ca06c5e6-4f50-4349-b753-7fe474bfbd14)

**Comment by Shelley-BaoYue** [comment]
"I solved the problem based on the methods provided online, but did not achieve the desired effect." 

>>  what does `did not achieve the desired effect` means? Do you still encounter this error log `Nameserver limits exceeded`? Or nameserver limits has been solved, but there are still other problems?

**Comment by kkkkeynoted** [comment]
I mean to say, this error `Nameserver limits exceeded` has been solved, but edgecore has not been added to the cloud yet. I don't know if this is caused by this issue`Nameserver limits exceeded` .Maybe I should try joining the cloud again.

**Comment by kkkkeynoted** [comment]
I want to rejoin the edge node in the cloud, but after I run `./keadm reset` ,an error has occurred `no such file`. Is it not necessary to reset keadm, but to delete edgecore？

**Comment by Shelley-BaoYue** [comment]
> I want to rejoin the edge node in the cloud, but after I run `./keadm reset` ,an error has occurred `no such file`. Is it not necessary to reset keadm, but to delete edgecore？

You could systemctl stop edgecore.service and then rejoin it.

**Comment by Shelley-BaoYue** [comment]
I will close this issue as it seems has been completed. If you have any questions, feel free to reopen it.
